apps/assisted_tagging/schema.py

```diff
@@ -205,9 +215,78 @@ def resolve_missing_prediction_reviews(root, info, **kwargs):
 
     @staticmethod
     def resolve_related_geoareas(root, info, **kwargs):
-        return root.related_geoareas.all()   # NOTE: Prefetched by DraftEntry
+        return root.related_geoareas.all()  # NOTE: Prefetched by DraftEntry
+
+
+class DraftEntryByLeadType(DjangoObjectType):
+    prediction_status = graphene.Field(DraftEntryPredictionStatusEnum, required=True)
+    prediction_status_display = EnumDescription(source='get_prediction_status_display', required=True)
+    predictions = graphene.List(
+        graphene.NonNull(AssistedTaggingPredictionType)
+    )
+    missing_prediction_reviews = graphene.List(
+        graphene.NonNull(MissingPredictionReviewType),
+    )
+    related_geoareas = graphene.List(
+        graphene.NonNull(ProjectGeoAreaType)
+    )
+    is_discarded = graphene.Int(required=True)
+
+    class Meta:
+        model = DraftEntry
+        only_fields = (
+            'id',
+            'excerpt',
+            'prediction_received_at',
+        )
+
+    @staticmethod
+    def custom_queryset(queryset, info, _filter, **kwargs):
+        return get_draft_entry_with_filter_qs(info, _filter).prefetch_related(
+            Prefetch(
+                'predictions',
+                queryset=AssistedTaggingPrediction.objects.order_by('id'),
+            ),
+            Prefetch(
+                'related_geoareas',
+                queryset=get_geo_area_queryset_for_project_geo_area_type().order_by('id'),
+            ),
+            'predictions__model_version',
+            'predictions__model_version__model',
+            'predictions__wrong_prediction_reviews',
+            'missing_prediction_reviews',
+            'related_geoareas',
+        )
+
+    @staticmethod
+    def resolve_predictions(root, info, **kwargs):
+        return root.predictions.filter(is_selected=True)   # NOTE: Prefetched by DraftEntry
+
+    @staticmethod
+    def resolve_missing_prediction_reviews(root, info, **kwargs):
+        return root.missing_prediction_reviews.all()   # NOTE: Prefetched by DraftEntry
+
+    @staticmethod
+    def resolve_related_geoareas(root, info, **kwargs):
+        return root.related_geoareas.all()
+
 
+class AutoextractionStatusType(graphene.ObjectType):
```

Comment:
```suggestion
class AutoExtractionStatusType(graphene.ObjectType):
```

Let's move this to the Lead Node as well.